common.mk

```diff
@@ -16,7 +16,7 @@ OPTEE_CLIENT_PATH		?= $(ROOT)/optee_client
 OPTEE_CLIENT_EXPORT		?= $(OPTEE_CLIENT_PATH)/out/export
 OPTEE_TEST_PATH			?= $(ROOT)/optee_test
 OPTEE_TEST_OUT_PATH		?= $(ROOT)/optee_test/out
-HELLOWORLD_PATH			?= $(ROOT)/hello_world
+EXAMPLES_PATH			?= $(ROOT)/optee_examples
```

Comment:
Minor: I would add an `OPTEE_` prefix for `EXAMPLES_PATH` and the other `EXAMPLES_...` below.